CustomItems/CustomItems.csproj

```diff
@@ -1,149 +1,141 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do not include changes to the csproj file that change the relative paths used for reference files.